pkg/nto/performanceprofile_test.go

```diff
@@ -41,6 +41,17 @@ var (
 			Node:  &defaultHugePagesNodeTwo,
 		},
 	}
+	defaultAnnotationOne = map[string]string{"performance.openshift.io/ignore-cgroups-version": "true"}
```

Comment:
This var use only once. Just copy paste map[string]string{"performance.openshift.io/ignore-cgroups-version": "true"} directly to the testCases.testAnnotations